msal/build/strong_signing.md

```diff
@@ -0,0 +1,17 @@
+### Strong-naming general notes
```

Comment:
>rong-naming general notes [](start = 6, length = 25)

also, the same info should be in the wiki or we should link to the md from the wiki... I suppose I prefer it being in the wiki over md files in general. then here we can reference the wiki page?